bootstrap/004_labels.sql

```diff
@@ -134,6 +134,102 @@ BEGIN
 
 END;
 
+CREATE OR REPLACE PROCEDURE INTERNAL.UPSERT_PREDEFINED_LABEL(name text, grp text, rank number, condition text)
+    RETURNS text
+    LANGUAGE SQL
+    EXECUTE AS OWNER
+AS
+BEGIN
+    if (:name is null) then
+      return 'Name must not be null.';
+    elseif (grp is null and rank is not null) then
+      return 'Rank must only be set if Group name is also provided.';
+    elseif (grp is not null and rank is null) then
+      return 'Rank must provided if you are creating a grouped label.';
+    end if;
+    let outcome text := 'Failure validating name & condition. Please check your syntax.';
+
+    outcome := (CALL INTERNAL.VALIDATE_LABEL_CONDITION(:name, :condition));
+
+    if (outcome is not null) then
+      return outcome;
+    end if;
+
+    outcome := (CALL INTERNAL.VALIDATE_LABEL_Name(:name));
+
+    if (outcome is not null) then
+      return outcome;
+    end if;
+
+    MERGE INTO internal.predefined_labels t
+    USING (SELECT :name as name, :grp as grp, :rank as rank, :condition as condition) s
+    ON t.name = s.name
+    WHEN MATCHED THEN
+    UPDATE
+        SET t.GROUP_NAME = s.grp, t.GROUP_RANK = s.rank, t.CONDITION = s.condition, t.LABEL_MODIFIED_AT = current_timestamp()
+    WHEN NOT MATCHED THEN
+    INSERT
+        ("NAME", "GROUP_NAME", "GROUP_RANK", "LABEL_CREATED_AT", "CONDITION", "LABEL_MODIFIED_AT")
+        VALUES (s.name, s.grp, s.rank,  current_timestamp(), :condition, current_timestamp());
+    outcome := null;
+
+    return outcome;
+END;
+
+CREATE OR REPLACE PROCEDURE INTERNAL.DELETE_PREDEFINED_LABEL(name text)
+    RETURNS text
+    LANGUAGE SQL
+    EXECUTE AS OWNER
+AS
+BEGIN
+    if (:name is null) then
+      return 'Name must not be null.';
+    end if;
+
+    DELETE FROM internal.predefined_labels where name = :name;
+    return 'done';
+END;
+
+--CREATE OR REPLACE PROCEDURE INTERNAL.MIGRATE_PREDEFINED_LABELS()
```

Comment:
please remove the commented proc

Reply:
removed.  this commented proc should be part of pr for migration (#110).